services/emailServices.js

```diff
@@ -3,29 +3,29 @@ const emailConfig = require("../config/email.config");
 const config = require("../config/config");
 
 const transport = nodemailer.createTransport({
-    host: emailConfig.SMTP_HOST,
-    port: emailConfig.SMPT_PORT,
-    auth: {
-        user: emailConfig.SMTP_USERNAME,
-        pass: emailConfig.SMTP_PASSWORD,
-    },
+  host: emailConfig.SMTP_HOST,
+  port: emailConfig.SMPT_PORT,
+  auth: {
+    user: emailConfig.SMTP_USERNAME,
+    pass: emailConfig.SMTP_PASSWORD,
+  },
 });
 
 const sendEmail = async (to, subject, text) => {
-    const msg = { from: emailConfig.EMAIL_FROM, to, subject, text };
-    await transport.sendMail(msg);
+  const msg = { from: emailConfig.EMAIL_FROM, to, subject, text };
+  await transport.sendMail(msg);
 };
 
 const sendVerificationEmail = async (to, token) => {
-    const subject = "Email Verification";
-    const verificationEmailUrl = `http://localhost:${config.port}/auth/v1/verify-email/${token}`;
-    const text = `Dear user,
+  const subject = "Email Verification";
+  const verificationEmailUrl = `http://localhost:${config.port}/auth/v1/verify-email/${token}`;
+  const text = `Dear user,
```

Comment:
create a file example emailTemplates and create a function pass verificationEmailUrl as argument and return string or html depends on template

Reply:
can we use templating engines here?

Reply:
not required as it will increase the bundle size.

Reply:
Resolved